blog/2024-09-04-lawn-mower-activity-returning.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Update the filename to todays date and then I think it's good to go 👍

Reply:
changed to 08-23